tests/phpunit/tests/rest-api/rest-settings-controller.php

```diff
@@ -83,13 +83,60 @@ public function test_context_param() {
 	public function test_get_item_is_not_public_not_authenticated() {
 		$request  = new WP_REST_Request( 'GET', '/wp/v2/settings' );
 		$response = rest_get_server()->dispatch( $request );
-		$this->assertSame( 401, $response->get_status() );
```

Comment:
Noting that for the devnote on this, I think there should be a mention that this URL no longer is a 401 for logged out visitors and it should include a code snipit to restore that if people want it